routes/boardsRouter.js

```diff
@@ -0,0 +1,62 @@
+const express = require("express");
+
+const ctrl = require("../controllers/boardsControllers");
+const { schemas } = require("../schemas/boardsSchemas");
+const {
+  validateBody,
+  isValidId,
+  boardWithoutId,
+  authenticate,
+  isCurrentUserOwner,
+} = require("../middlewares");
+
+const boardsRouter = express.Router();
+
+boardsRouter.get("/", authenticate, ctrl.getAllBoards);
+
+boardsRouter.get(
+  "/:boardId",
+  // isValidId,
+  // boardWithoutId,
+  authenticate,
+  // isCurrentUserOwner,
+  ctrl.getById
+);
+
+boardsRouter.delete(
+  "/:boardId",
+  isValidId,
+  boardWithoutId,
+  authenticate,
+  isCurrentUserOwner,
+  ctrl.deleteBoard
+);
+
+boardsRouter.post(
+  "/",
+  validateBody(schemas.createBoardSchema),
+  authenticate,
+  ctrl.createBoard
+);
+
+boardsRouter.put(
+  "/:boardId",
+  isValidId,
```

Comment:
We really need put methods if we already have patch?

Reply:
may be dont....